ci/base-images/alpine/Dockerfile.java21

```diff
@@ -21,9 +20,9 @@ RUN apk update && apk add --no-cache \
     make \
     git \
     && npm install -g corepack \
-    && /usr/bin/python${PYTHON_VERSION} --version \
-    && /usr/bin/python${PYTHON_VERSION} -m pip install --break-system-packages --no-cache-dir --upgrade setuptools==77.0.3 wheel pip virtualenv \
-    && /usr/bin/python${PYTHON_VERSION} -m pip install --break-system-packages --no-cache-dir --upgrade pipenv atom-tools --target /opt/pypi \
+    && /usr/bin/python --version \
+    && /usr/bin/python -m pip install --break-system-packages --no-cache-dir --upgrade setuptools==77.0.3 wheel pip virtualenv \
```

Comment:
We don't need setuptools pip etc in java images. We need them only in python images.